src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_transformers.py

```diff
@@ -11,10 +11,40 @@ def transform_usage_output(result):
     result.billable_quantity = str(result.billable_quantity)
     result.pretax_cost = str(result.pretax_cost)
     if result.meter_details:
-        result.meter_details.total_included_quantity = str(result.meter_details.total_included_quantity)
-        result.meter_details.pretax_standard_rate = str(result.meter_details.pretax_standard_rate)
+        result.meter_details.total_included_quantity = str(
+            result.meter_details.total_included_quantity)
+        result.meter_details.pretax_standard_rate = str(
+            result.meter_details.pretax_standard_rate)
```

Comment:
Does it not fit on one line?

Reply:
its result of automatic format feature of Visual Studio Code

Reply:
corrected formating